gpfit/print_fit.py

```diff
@@ -10,10 +10,10 @@ def print_ISMA(A, B, alpha, d, K):
             print printString
             printString = '    + '
 
-        printString += '({0:.3g}/w**{1:.3g})'.format(exp(alpha[k] * B[k]), alpha[k])
```

Comment:
Yah, my bad. I can(/will) change it, but remind me why that convention was chosen? x and y seem like more natural choices for normal-space independent and dependent variables.